nixos/tests/docker-tools.nix

```
@@ -419,5 +419,10 @@ import ./make-test-python.nix ({ pkgs, ... }: {
             "docker rmi layered-image-with-path",
         )
 
+    with subtest("etc"):
+        docker.succeed("${examples.etc} | docker load")
```

Does this not need to be

```suggestion
        docker.succeed("cat ${examples.etc} | docker load")
```

or

```suggestion
        docker.succeed("docker load -i ${examples.etc}")
```

?

---

ahh